packages/plugin-ext/src/common/index.ts

```diff
@@ -0,0 +1,8 @@
+/*
+ * Copyright (C) 2018 Red Hat, Inc. and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+export * from '../hosted/node/hosted-plugin-uri-postprocessor';
```

Comment:
Maybe some doc about what it provides to other modules
And in readme as well

Reply:
ok